src/brevitas/graph/equalize.py

```diff
@@ -55,6 +57,8 @@
     operator.imul,
     operator.__mul__,
     operator.__imul__,
+    operator.getitem,
```

Comment:
Test with torch symbolic tracer if this op is needed. In case move to _select_op category